packages/slider/src/HandleController.ts

```diff
@@ -341,6 +341,22 @@ export class HandleController {
 
     private _activePointerEventData!: DataFromPointerEvent | undefined;
 
+    /**
+     * @description check for defaultvalue(value) property in sp-slider and reset on double click on sliderHandle
+     * @param event
+     */
+    public handleDoubleClick(event: PointerEvent): void {
+        const input = (event.target as Element).querySelector(
+            '.input'
+        ) as InputWithModel;
+
+        if (input.model?.handle.defaultValue !== undefined) {
+            input.model.handle.value = input.model.handle.defaultValue;
+            this.dispatchChangeEvent(input, input.model.handle);
```

Comment:
Confirm whether this dispatches an `input` event with the same data. See the result of clicking a new value in an `<input type="range">` as an example: https://codepen.io/Westbrook/pen/ExJjVRZ?editors=1111

Reply:
added  `dispatchInputEvent` on both double-click and escape key interactions

Reply:
What's a good test to confirm that this happens before we close this up?

Reply:
would it make sense if I extend the `dispatches input of the animation frame` test (line 388) or should I make a separate one?